traffic_ops/testing/api_contract/v4/conftest.py

```diff
@@ -471,3 +471,52 @@ def cachegroup_post_data(to_session: TOSession, request_template_data: list[JSON
 	except IndexError:
 		logger.error("No Cache group response data from cdns POST request.")
 		sys.exit(1)
+
+
+@pytest.fixture()
+def role_post_data(to_session: TOSession, request_template_data: list[JSONData]
+			) -> dict[str, object]:
+	"""
+	PyTest Fixture to create POST data for roles endpoint.
+
+	:param to_session: Fixture to get Traffic Ops session.
+	:param api_prerequisite_data: Fixture to get role data from a prerequisites file.
```

Comment:
should be `request_template_data` not `api_prerequisite_data`

Reply:
Updated it!